frontend/integration-tests/views/devconsole-view/git-imort-flow.ts

```diff
@@ -9,7 +9,10 @@ export const applicationDropdown = element(
 );
 export const createApplication = element(by.id('#CREATE_APPLICATION_KEY#-link'));
 export const applicationName = element(by.css('[data-test-id="application-form-app-input"]'));
-export const appName = element(by.css('[data-test-id="application-form-app-name"]'));
+
+//export const appName = element(by.css('[data-test-id="application-form-app-name"]'));
+export const appName = element(by.xpath('//*[@id="form-input-name-field"]'));
```

Comment:
@ldimaggi we are avoiding the use of x-path, therefore, I've used data-test-id

Reply:
Agreed - the use of xpath has been removed.